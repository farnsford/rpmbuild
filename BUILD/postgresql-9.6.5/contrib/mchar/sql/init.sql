
--
-- first, define the datatype.  Turn off echoing so that expected file
-- does not depend on contents of mchar.sql.
--

\set ECHO none
\i mchar.sql
--- load for table based checks
SET search_path = public;
\set ECHO all

create table ch (
	chcol mchar(32)
) without oids;

insert into ch values('abcd');
insert into ch values('AbcD');
insert into ch values('abcz');
insert into ch values('defg');
insert into ch values('dEfg');
insert into ch values('ee');
insert into ch values('Ee');

create table chvch (
    ch      mchar(12),
	vch     mvarchar(12)
) without oids;

insert into chvch values('No spaces', 'No spaces');
insert into chvch values('One space ', 'One space ');
insert into chvch values('1 space', '1 space ');

