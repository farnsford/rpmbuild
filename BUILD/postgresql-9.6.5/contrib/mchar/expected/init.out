--
-- first, define the datatype.  Turn off echoing so that expected file
-- does not depend on contents of mchar.sql.
--
\set ECHO none
psql:mchar.sql:20: NOTICE:  type "mchar" is not yet defined
DETAIL:  Creating a shell type definition.
psql:mchar.sql:25: NOTICE:  argument type mchar is only a shell
psql:mchar.sql:30: NOTICE:  argument type mchar is only a shell
psql:mchar.sql:35: NOTICE:  return type mchar is only a shell
psql:mchar.sql:59: NOTICE:  type "mvarchar" is not yet defined
DETAIL:  Creating a shell type definition.
psql:mchar.sql:64: NOTICE:  argument type mvarchar is only a shell
psql:mchar.sql:69: NOTICE:  argument type mvarchar is only a shell
psql:mchar.sql:74: NOTICE:  return type mvarchar is only a shell
create table ch (
	chcol mchar(32)
) without oids;
insert into ch values('abcd');
insert into ch values('AbcD');
insert into ch values('abcz');
insert into ch values('defg');
insert into ch values('dEfg');
insert into ch values('ee');
insert into ch values('Ee');
create table chvch (
    ch      mchar(12),
	vch     mvarchar(12)
) without oids;
insert into chvch values('No spaces', 'No spaces');
insert into chvch values('One space ', 'One space ');
insert into chvch values('1 space', '1 space ');
